rampwf/prediction_types/detection.py

```diff
@@ -243,20 +243,36 @@ def _sample_predictions(preds, n_samples=100):
         return []
 
 
+def _find_mixture_gaussians_kmean(data, n_clusters, max_n_samples):
+    from sklearn.cluster import KMeans
+    km = KMeans(n_clusters=n_clusters)
+    labels = km.fit_predict(data)
+
+    objs = []
+
+    for i in range(n_clusters):
+        mask = labels == i
+        c = mask.sum() / max_n_samples
+        x, y = km.cluster_centers_[i]
+        r = np.mean((data[mask] - km.cluster_centers_[i])**2)
+        objs.append((c, x, y, r))
```

Comment:
Why the choice of average of squares instead of the Euclidean distance ?

Reply:
You mean for the calculation of the radius `r` ? Because I initially used the radius as the variance of multivariate normal that I sampled from. So therefore here using that as an estimate of the radius.